cmd/root.go

```diff
@@ -84,36 +84,42 @@ func sendEmail(to, subject, body string) error {
 	return gomail.Send(s, m)
 }
 
-func sendEmails(matches []MatchPair) {
+func sendEmails(matches []MatchPair) ([]string, error) {
+	// TODO: add retry logic for failed emails (maybe use a queue)
+	// instead of sending each email individually,
+	// generate all the emails and send them in bulk to avoid individual errors
+	emailIssues := []string{}
```

Comment:
You could pre-allocate the capacity of this. Insignificant micro-optimization in this case, but good habit.

```go
emailIssues := make([]string, 0, len(matches))
```

I think the above is correct, but untested